### DISCUSSION TITLE
CZM examples

### DISCUSSION THREAD

**Discussion by ghost** [idaholab/moose#25670] [Q&A Modules: Solid mechanics]
Hello, 

I've noticed that the examples for the cohesive zone modeling do not work. I've tried to fix it up but it's still not working, I do not see where the interface boundary should be as in the other working files (ad_czm or bilinear_mixed_scale_strength) they are not mentioned either. 

Thank you. 
[czm_multiple_dimension_base.txt](https://github.com/idaholab/moose/files/12820054/czm_multiple_dimension_base.txt)

**Comment by GiudGiud** [comment]
Hello

They work fine, it s just that you need to use the command line arguments in the `tests` file. 

What the developers did here is that they created the skeleton for a simulation, and in the tests pass the missing pieces to make it work through the command line.

It's not ideal but it limits our maintenance burden.
When we have (in development)
- input file includes
- ability to delete existing parameters from the command line

We can make the test files work on their own and re-use them for several tests

Guillaume
